pkg/debug/scripts/collect-info.sh

```diff
@@ -330,12 +331,30 @@ collect_pillar_backtraces
 # ZFS part
 collect_zfs_info
 
+check_tar_flags() {
```

Comment:
@OhmSpectator , isn't enough to check the versions?

Busybox tar:
```
c6199ba3-6a13-409b-a3a5-697264c6a4bb:~# tar --version
tar (busybox) 1.35.0
```

For GNU tar (which supports the flags):
```
900471d3-e720-4e0e-ba06-6978f635dbb9:~# tar --version
tar (GNU tar) 1.34
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
```

Perhaps `tar --version | grep "GNU tar"` does the job...